client/extensions/woocommerce/woocommerce-services/views/label-settings/index.js

```diff
@@ -1,109 +1,110 @@
 import ExtendedHeader from 'woocommerce/components/extended-header';
-import FormFieldSet from 'components/forms/form-fieldset';
-import FormLabel from 'components/forms/form-label';
-import FormSelect from 'components/forms/form-select';
 import FormToggle from 'components/forms/form-toggle';
-import ShippingCard from './shipping-card';
+import LabelSettings from './label-settings';
+import * as settingsActions from '../../state/label-settings/actions';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import {
+	getLabelSettingsFormData,
+	getLabelSettingsFormMeta,
+	getLabelSettingsStoreOptions
+} from '../../state/label-settings/selectors';
 
-class LabelSettings extends Component {
-	constructor( props ) {
-		super( props );
+class AccountSettingsRootView extends Component {
 
-		//TODO: use redux state with real data
-		this.state = {
-			visible: true,
-			cards: [ {
-				selected: true,
-				type: 'VISA',
-				digits: '1234',
-				name: 'Name Surname',
-				date: '12/19'
-			}, {
-				selected: false,
-				type: 'MasterCard',
-				digits: '5678',
-				name: 'Name Surname',
-				date: '01/20'
-			} ]
-		};
+	componentWillMount() {
+		if ( this.props.siteId ) {
+			this.props.actions.fetchSettings( this.props.siteId );
+		}
 	}
 
-	selectCard( index ) {
-		const cards = this.state.cards.map( ( card ) => {
-			return { ...card, selected: false };
-		} );
-
-		cards[ index ].selected = true;
-
-		this.setState( { cards } );
+	componentWillReceiveProps( props ) {
+		if ( props.siteId !== this.props.siteId ) {
+			this.props.actions.fetchSettings( props.siteId );
+		}
 	}
 
 	render() {
-		const { translate } = this.props;
+		const { formData, formMeta, storeOptions, siteId, translate, actions } = this.props;
 
-		const onToggle = () => {
-			this.setState( { visible: ! this.state.visible } );
-		};
+		if ( ! formMeta ) {
+			return null;
+		}
+		const setFormDataValue = ( key, value ) => ( actions.setFormDataValue( siteId, key, value ) );
+		const onEnabledToggle = () => ( actions.setFormDataValue( siteId, 'enabled', ! formData.enabled ) );
 
-		const renderCard = ( card, index ) => {
-			const onSelect = () => {
-				this.selectCard( index );
-			};
+		const renderContent = () => {
+			if ( ! formData && ! formMeta.isFetching ) {
+				return (
+					<p>
+						{ translate( 'Unable to get your settings. Please refresh the page to try again.' ) }
+					</p>
+				);
+			}
 
-			return ( <ShippingCard
-				key={ index }
-				onSelect={ onSelect }
-				{ ...card } /> );
+			return (
+				<LabelSettings
+					isLoading={ formMeta.isFetching }
+					pristine={ formMeta.pristine }
+					paymentMethods={ formMeta.payment_methods || [] }
+					setFormDataValue={ setFormDataValue }
+					selectedPaymentMethod={ ( formData || {} ).selected_payment_method_id }
+					paperSize={ ( formData || {} ).paper_size }
+					storeOptions={ storeOptions }
+				/>
+			);
 		};
 
+		const renderToggle = formData && isBoolean( formData.enabled );
```

Comment:
This might be worth a comment like the one you left in the PR notes:
`hide the toggle when the enabled flag is not present (older version of WCS) and to respect it otherwise.`